### DISCUSSION TITLE
GitHub Copilot support with delphi IDE

### DISCUSSION THREAD

**Discussion by rachna2512** [community/community#64463] [Copilot Conversations]
### Select Topic Area

General

### Body

Can we install and use Github copilot with delphi 10.2 IDE? If yes, is there any documentation available for it.

**Comment by mehulrajdev** [comment]
Read this I guess
https://docs.github.com/en/copilot/getting-started-with-github-copilot
https://github.com/ahausladen/DDevExtensions

**Reply by Rockyread** [reply]
discussions

**Comment by BoscoBecker** [comment]
I need that so much!!!

**Reply by Thuem26** [reply]
Hiển

**Comment by ads69** [comment]
I think it's time for embarcadero to avoid loosing time with their IDE, they should focus on seling a good Pascal/Delphi compiler (like it is now) and release a plugin (including the designer part) for VS Code.

**Comment by plamenjm** [comment]
Delphi/pascal file in IntelliJ IDEA and Copilot suggestions.

With I-Pascal plugin for IntelliJ (version before 2022.1 !!)
https://plugins.jetbrains.com/plugin/7340-i-pascal

In the editor on the left side - suggested auto-complete
On the right side - copilot panel with multiple suggestions

1. Without hint
![image](https://github.com/community/community/assets/56994434/81836842-df05-4d74-8cec-17874ac54556)


2. With hint
![image](https://github.com/community/community/assets/56994434/0eb33573-a467-499b-8c50-3072556e8922)
